content/blog/2020/2020-04-02-covid-19-basic.md

```diff
@@ -0,0 +1,205 @@
++++
+author = "Janez Demšar"
+date = "2020-03-02"
```

Comment:
Make it 2020-04-02 to have the right sorting.

Reply:
But only just before publishing, otherwise my local hugo doesn't show it.